build/grpc/Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM ${BUILDARCH}-base
 ARG PROTOC_VERSION=3.18.0
 ARG PROTOC_ZIP=protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip
 ARG GRPC_WEB_VERSION=1.3.0
-ARG GATEWAY_VERSION=2.6.0
+ARG GATEWAY_VERSION=2.15.1
```

Comment:
I needed to update the grpc-gateway to mitigate this error https://github.com/zitadel/zitadel/actions/runs/4247961435/jobs/7386579558#step:7:241
@livio-a @hifabienne  i hope this does not break anything